Automattic/wp-calypso
# Launchpad: Warn if global styles used but site incapable

#### Proposed Changes

Make the launchpad mark the plan step as incomplete if global styles are being used by the site, but the site isn't eligible to use them.

#### Testing Instructions



1. Use the live link or whatever
2. Go to /setup/newsletter
3. Go through the process choosing free domain, plan etc
4. After the 'Adding subscribers' screen you should see a preview of your site and some steps to the left. This should be the only screen affected by this PR
6. Note that 'Choose a plan' shows your chosen plan
7. Note that 'Choose a plan' looks done (crossed out, ticked off)
8. In another tab, apply the `wpcom-limit-global-styles` sticker to the site
9. Go back to the previous page and reload the site

 * If you are on personal plan or higher then nothing should change - the screen looks done
 * If you are on any plan and have chosen the default 'accent colour' then nothing should change - the screen looks done
 * The screen looks like this: 
 * If you are on free and have chosen a non-default 'accent colour' then you should see a not-done screen. Clicking anywhere on Choose a Plan should take you to the plans page with Premium highlighted

  Done | Not-done
-------|---------
![image](https://user-images.githubusercontent.com/93301/205698466-0bcbd87a-fa5a-4407-8db6-630eddd0c9f3.png) | ![image](https://user-images.githubusercontent.com/93301/205700025-898065dc-c7a9-424e-95de-8e9f33305fef.png)

Other flows such as VideoPress (/setup/videopress) should be unaffected.



#### Pre-merge Checklist





Related to https://github.com/Automattic/dotcom-forge/issues/1227
